src/hotspot/share/opto/cfgnode.cpp

```diff
@@ -3109,6 +3109,9 @@ void NeverBranchNode::format( PhaseRegAlloc *ra_, outputStream *st) const {
 }
 #endif
 
+Node* BlackholeNode::Ideal(PhaseGVN* phase, bool can_reshape) {
+  return remove_dead_region(phase, can_reshape) ? this : nullptr;
+}
```

Comment:
I think you need a newline after this definition.

Reply:
Done.